form.php

```diff
@@ -0,0 +1,145 @@
+<?php
+    $fullName = "";
+    $trimmed = "";
+    $namePattern = '/^[a-zA-Z]+$/';
+    if ($_SERVER['REQUEST_METHOD'] == 'POST' && isset($_POST['Submit'])) {
+        $fName = $_POST['fName'];
+        $lName = $_POST['lName'];
+        $fullName = $fName." ".$lName;
+        $trimmed = preg_replace($namePattern, " ", $fullName);
+    }
+?>
+<?php
```

Comment:
Why have you closed the php tag and opened it again?

Reply:
Correction done.